src/components/inviteWelcome/InviteWelcome.js

```diff
@@ -10,53 +10,26 @@ import { IS_FIRST_VISIT } from '../../lib/constants/localStorage'
 import CustomButton from '../common/buttons/CustomButton'
 import Wrapper from '../common/layout/Wrapper'
 import Text from '../common/view/Text'
-import WalletSVG from '../../assets/Invite/wallet.svg'
-import SelfieSVG from '../../assets/Invite/selfie.svg'
-import MobileSVG from '../../assets/Invite/mobile.svg'
 
-import Section from '../common/layout/Section'
-import { getDesignRelativeHeight, isShortDevice, isVeryShortDevice } from '../../lib/utils/sizes'
-import NavBar from '../appNavigation/NavBar'
-import { theme } from '../theme/styles'
+import {
```

Comment:
please revert this component back

add a new component in src/compontent/auth instead, name it Welcome